### ISSUE TITLE
Timestamp

### ISSUE THREAD

**Issue by dapolinario** [Ettercap/ettercap#718]
I need to get the timestamp of the package that is going through the MITM to deal with the etterfilter, but I did not find this variable. It exists ? I tried to use timeval and got error in the compilation of etterfilter.

**Comment by koeppea** [comment]
Hi,
you mean the TCP timestamp option defined with RFC1323?
No they're not dissected and provided to the filter engine.

**Comment by dapolinario** [comment]
No. What I need is the timestamp for logging purposes, knowing the moment ettercap received the package and not the timestamp of the package as documented in RFC.
